cosmo_tester/resources/blueprints/component/fake_infra.yaml

```diff
@@ -0,0 +1,29 @@
+tosca_definitions_version: cloudify_dsl_1_3
```

Comment:
You can use cosmo_tester/resources/blueprints/capabilities/fake_vm.yaml instead of this I think?

Reply:
It might be worth merging the directories into either of capabilities or component (no judgement inherent in order, just a lexical sort)?